CHANGELOG.md

```diff
@@ -1,20 +1,53 @@
 # Azure Kubernetes Service Changelog
 
+
+
+## Release 2023-08-20
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
```

Comment:
To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023
on clusters that haven’t specified an update option explicitly, mapping to the option “
None
” in the
[node OS upgrade](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fazure%2Faks%2Fauto-upgrade-node-image&data=05%7C01%7Cv-gapowe%40microsoft.com%7C3cc47a530db74dae8a9208dba50c4559%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638285244344728560%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=DBilgB0Cpavy9j4qXezu5bW6cSwwIdYhH2Y9U2hlCDk%3D&reserved=0)
channel feature. AKS strongly recommends using and proactively moving to 
[auto-upgrade node-image](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fazure%2Faks%2Fauto-upgrade-cluster&data=05%7C01%7Cv-gapowe%40microsoft.com%7C3cc47a530db74dae8a9208dba50c4559%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638285244344728560%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ua3xhPdQsw96P%2Bbvhdurl6uZkx053fzYzl%2BdLwwttxs%3D&reserved=0)
 or 
[node OS upgrade channel - SecurityPatch](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fazure%2Faks%2Fauto-upgrade-node-image&data=05%7C01%7Cv-gapowe%40microsoft.com%7C3cc47a530db74dae8a9208dba50c4559%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638285244344728560%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=DBilgB0Cpavy9j4qXezu5bW6cSwwIdYhH2Y9U2hlCDk%3D&reserved=0)
; you can set 
[maintenance windows](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fazure%2Faks%2Fplanned-maintenance&data=05%7C01%7Cv-gapowe%40microsoft.com%7C3cc47a530db74dae8a9208dba50c4559%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638285244344728560%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=M1XlSRNaAHBUQbRcoQErkqxGuqVF78jEZUVOoaWhhsc%3D&reserved=0)
 for these channels.